examples/linear_array/color_doppler/ops.py

```diff
@@ -159,45 +162,12 @@ def __init__(self, wn, n, ftype='butter', btype='highpass'):
 
     def prepare(self, metadata):
 
-        if self.ftype == 'butter':
-            self.ba = scipy.signal.iirfilter(
-                self.n,
-                self.wn,
-                ftype=self.ftype,
-                btype=self.btype,
-                output='ba',
-            )
-        elif self.ftype == 'cheby1':
-            self.ba = scipy.signal.iirfilter(
-                self.n,
-                self.wn,
-                rp=10,
-                rs=100,
-                ftype=self.ftype,
-                btype=self.btype,
-                output='ba',
-            )
-        elif self.ftype == 'cheby2':
-            self.ba = scipy.signal.iirfilter(
-                self.n,
-                self.wn,
-                rp=10,
-                rs=100,
-                ftype=self.ftype,
-                btype=self.btype,
-                output='ba',
-            )
-        elif self.ftype == 'ellip':
-            self.ba = scipy.signal.iirfilter(
-                self.n,
-                self.wn,
-                rp=10,
-                rs=100,
-                ftype=self.ftype,
-                btype=self.btype,
-                output='ba',
-            )
-        elif self.ftype == 'bessel':
+        if (   self.ftype == 'butter'
+            or self.ftype == 'cheby1'
+            or self.ftype == 'cheby2'
+            or self.ftype == 'ellip'
+            or self.ftype == 'bessel'):
```

Comment:
This condition can be simplified to:
```
if self.ftype in {"butter", "cheby1", "cheby2", "ellip", "bessel"}:
```
Also, please use the double quote for strings (").